src/app/admin/projects/[project_id]/chats/[chat_id]/page.tsx

```diff
@@ -27,6 +28,36 @@ export default function Page({
   const project: Project = projectQuery.data;
   const chats: any | undefined = chatQuery.data;
 
+  const feedbackQuery = useQuery({
+    queryKey: ["project", project_id, "chat", chat_id],
+    queryFn: () => API.feedback.list(project_id, chat_id),
+  });
```

Comment:
Can you add the expected type of data here? `useQuery<DataType>({...`

Reply:
@skks1212 there is no existing DataType declared for useQuery response, right? Should I just declare a new one? Also, the backend PR was merged but the staging hasn't been updated, could you refresh that.